crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/ClipSurfaceIncludeArea.shader

```diff
@@ -0,0 +1,49 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// Renders the geometry to the clip surface data and sets the value to 'include'.
+
+Shader "Crest/Inputs/Clip Surface/Include Area"
+{
+	SubShader
+	{
+		Tags { "Queue" = "Geometry-10" }
+
+		Pass
+		{
+			Blend Off
+			ZWrite Off
+			ColorMask R
+
+			CGPROGRAM
+			#pragma vertex Vert
+			#pragma fragment Frag
+
+			#include "UnityCG.cginc"
+
+			struct Attributes
+			{
+				float3 positionOS : POSITION;
+			};
+
+			struct Varyings
+			{
+				float4 positionCS : SV_POSITION;
+			};
+
+			Varyings Vert(Attributes input)
+			{
+				Varyings o;
+				o.positionCS = UnityObjectToClipPos(input.positionOS);
+				return o;
+			}
+
+			half Frag(Varyings input) : SV_Target
```

Comment:
Changing half to half4 will clear an error on MacOS raised by the material preview:
```
Metal: Error creating pipeline state (Crest/Inputs/Clip Surface/Include Area): fragment shader color output does not have enough components for the pixel format (MTLPixelFormatRGBA16Float)
```

Reply:
Good catch, thanks, fixed